modules/virt-runbook-kubevirtnoavailablenodestorunvms.adoc

```diff
@@ -0,0 +1,81 @@
+// Do not edit this module. It is generated with a script.
+// Do not reuse this module. The anchor IDs do not contain a context statement.
+// Module included in the following assemblies:
+//
+// * virt/support/virt-runbooks.adoc
+
+:_content-type: REFERENCE
+[id="virt-runbook-KubeVirtNoAvailableNodesToRunVMs"]
+= KubeVirtNoAvailableNodesToRunVMs
+
+[discrete]
+[id="meaning-kubevirtnoavailablenodestorunvms"]
+== Meaning
+
+This alert fires when the node CPUs in the cluster do not support virtualization
+or the virtualization extensions are not enabled.
+
+[discrete]
+[id="impact-kubevirtnoavailablenodestorunvms"]
+== Impact
+
+The nodes must support virtualization and the virtualization features must be
+enabled in the BIOS to run virtual machines (VMs).
+
+[discrete]
+[id="diagnosis-kubevirtnoavailablenodestorunvms"]
+== Diagnosis
+
+* Check the nodes for hardware virtualization support:
++
+[source,terminal]
+----
+$ oc get nodes -o json|jq '.items[]|{"name": .metadata.name, "kvm": .status.allocatable["devices.kubevirt.io/kvm"]}'
+----
++
+.Example output
++
+[source,text]
+----
+{
+  "name": "shift-vwpsz-master-0",
+  "kvm": null
+}
+{
+  "name": "shift-vwpsz-master-1",
+  "kvm": null
+}
+{
+  "name": "shift-vwpsz-master-2",
+  "kvm": null
+}
+{
+  "name": "shift-vwpsz-worker-8bxkp",
+  "kvm": "1k"
+}
+{
+  "name": "shift-vwpsz-worker-ctgmc",
+  "kvm": "1k"
+}
+{
+  "name": "shift-vwpsz-worker-gl5zl",
+  "kvm": "1k"
+}
+----
++
+Nodes with `"kvm": null` or `"kvm": 0` do not support virtualization extensions.
++
+Nodes with `"kvm": "1k"` do support virtualization extensions
```

Comment:
seems like this would be better suited as code callouts

Reply:
Can't be done. These files are generated automatically from Markdown files and Markdown does not support code callouts.